polkadot/Cargo.toml

```diff
@@ -115,3 +115,7 @@ metadata-hash = [
 # Enables timeout-based tests supposed to be run only in CI environment as they may be flaky
 # when run locally depending on system load
 ci-only-tests = ["polkadot-node-core-pvf/ci-only-tests"]
+
+experimental-collator-protocol = [
```

Comment:
Let's keep this feature enabled by default to make the testing easier.

The implementation is already behind a CLI flag, it's pretty self-contained. I think it should be safe. Any objections?

Reply:
Rather than enable this feature all the time, just remove it.